main_utils.py

```diff
@@ -25,6 +25,10 @@ async def on_channel_create(self, channel):
         """
         EXTENSION 1 + 3: Staff notes for tickets! and button stripping.
         Minor Extension 2: Safe Community Support
+        
+        This event works based on the ticket_users configuration.
+        Please ensure you add your ticket bot ID or approved
+         ticket invoking user IDs to the config file or via commands.
```

Comment:
FOR FECKS SAKE FORMATTING

Reply:
THE STUPID FECKING DEEPSOURCE DOESNT LIKE HOW LONG OF A SENTENCE IT IS SO I INDENTED IT WHAT DO YOU WANT FROM ME HJKDKASKDH

Reply:
THE SPACE

Reply:
> THE STUPID FECKING DEEPSOURCE DOESNT LIKE HOW LONG OF A SENTENCE IT IS SO I INDENTED IT WHAT DO YOU WANT FROM ME HJKDKASKDH
You could at least allign the lines :(

Reply:
AT THE BEGGINING

Reply:
*points gun*

Reply:
I did it BECAUSE IT WAS TOO LONG AND SHOWS ITS A CONTINUATION SUGMANUTS

Reply:
fine I'll change it but don't say I didn't try smh

Reply:
@benfoster04 please, you are scaring the children with your dangling space